### ISSUE TITLE
Relative timestamp for last update

### ISSUE THREAD

**Issue by unoukujou** [martykan/forecastie#668]
Example:

**Last update: 6:25AM**

Can be relative like this:

**Updated just now
Updated 30 seconds ago
Updated 1 minute ago
Updated 5 minutes ago
Updated 10 minutes ago
Updated 15 minutes ago
Updated 20 minutes ago
Updated 25 minutes ago
Updated ... minutes ago
Updated 1 hour ago
Updated 2 hours ago
Updated ... hours ago
Updated over a day ago**

Something like that makes it easy so you don't have to glance at the clock to check how long it has been since the last update.

**Comment by robinpaulson** [comment]
Yes, this is a good idea. It hurts my brain trying to process timestamps sometimes, partic if they're from before midnight.

**Comment by unoukujou** [comment]
Same thing for me lol. The "Last Update" timestamp is just not enough information because after you still need to check the current time, then do some thinking to figure it out. And this one little timestamp is the first thing I'm constantly looking at every time I open Forecastie because it makes all the rest of the data more relevant when you know it's fresh or not. So you can imagine.

**Comment by robinpaulson** [comment]
I think OWM initiates a new forecast every 6 hours, so I'm tempted to go for less precision than you suggested. That's detail of course, and shouldn't detract from the implementation. Maybe:
Updated in the last hour
Updated in the last 3 hours
Updated in the last 6 hours
Updated in the last day
Updated over a day ago
It might also reduce obsessive updating, which is something I've done in the past. That in turn will mean fewer hits on the OWM server and thus fewer counts against the total number we are permitted on the FLOSS subscription plan.
I might think about the frequency with which it auto-updates as well, there are more than necessary in the settings page at the moment.

**Comment by unoukujou** [comment]
Yes, that works for me just fine like that. The main thing for me is just to be able to know I'm looking at recent data from just one glance instead of needing a second look at the current time. Saves those few extra seconds and makes it more pleasant. Since I started using Forecastie, it's the one thing that always drives me crazy. It's just one of those things.

**Comment by robinpaulson** [comment]
> Saves those few extra seconds

over a lifetime that must add up to several minutes

**Comment by unoukujou** [comment]
Now that you mention that, could be several minutes in just a month or two. Looking at the timestamp, looking at the time, thinking if everything is good... say it takes 2-3 seconds. Checking Forecastie around 3 times a day, 6-9 seconds a day (42-63 seconds per week).
